buildh/core.py

```diff
@@ -142,166 +205,85 @@ def build_all_Hs_calc_OP(universe_woH, dic_lipid, dic_Cname2Hnames,
         This dict should contain the correspondance between the numres and
         the corresponding index in dic_OP. For example {..., 15: 14, ...} means
         the residue numbered 15 in the PDB has an index of 14 in dic_OP.
-    return_coors : boolean (optional)
-        If True, the function will return a pandas dataframe containing the
-        system *with* hydrogens.
-
-    Returns
-    -------
-    pandas dataframe (optional)
-        If parameter return_coors is True, this dataframe contains the
-        system *with* hydrogens is returned.
-    None
-        If parameter return_coors is False.
     """
-    if universe_wH:
-        # We will need the index in the numpy array for updating coordinates
-        # in the universe with H.
-        row_index_coor_array = 0
-    if return_coors:
-        # The list newrows will be used to store the new molecule *with* H.
-        newrows = []
-        # Counter for numbering the new mlcs with H.
-        new_atom_num = 1
+    # We will need the index in the numpy array for updating coordinates
+    # in the universe with H.
+    row_index_coor_array = 0
+    resid = -9999
```

Comment:
Why this initialization? I don't understand. Ha now I understand that's because you want it to be different when you do the test `if resid != atom.residue.resid:` for the first atom.

Reply:
yes!